### ISSUE TITLE
Update Chart.js support

### ISSUE THREAD

**Issue by blasss** [HtmlUnit/htmlunit#802]
HTMLUnit currently includes support for Chart.js version 2.9.4 (https://github.com/HtmlUnit/htmlunit/tree/master/src/test/resources/libraries/chartjs/2.9.4) but the latest version is currently 4.4.3 (https://www.chartjs.org/docs/latest/api/).

Using the latest version of Chart.js should be possible, but fails with a JavaScript error ("redeclaration of const..."), that seems to be related to a rhino bug (https://github.com/mozilla/rhino/issues/326).

**Comment by blasss** [comment]
I saw a [commit ](https://github.com/HtmlUnit/htmlunit-rhino-fork/commit/82c50412fe575a9240d6ddc56a18a8cd776cabad) related to the mentioned "redeclaration of const..." in the latest 4.3.0 release and gave it another try.
Unfortunately it is still failing. This time it seems to be related to using default parameters ("missing ) after formal parameters")...

**Comment by rbri** [comment]
@blasss Yes i know (i also did a quick test) but there is hope for this problem also; see https://github.com/mozilla/rhino/pull/1481
